Source/WebCore/platform/BoxSides.h

```diff
@@ -119,18 +119,28 @@ constexpr LogicalBoxCorner mapCornerPhysicalToLogical(WritingMode, BoxCorner);
 
 /** Implementation Below **********************************************/
 
+constexpr BoxAxis WritingMode::blockAxis() const
+{
+    return static_cast<BoxAxis>(isHorizontal());
```

Comment:
A ternary would be less risky / more robust than a static_cast

Reply:
I am no CSS person but I also find it confusing that `isHorizontal() == true` seems to translate to `BoxAxis::Vertical` and not `BoxAxis::Horizontal` 🤷🏻

Reply:
Note that clang generates the same assembly code for both functions here:
```
enum class BoxAxis : bool {
    Vertical,
    Horizontal
};

BoxAxis blockAxis(bool isHorizontal)
{
    return static_cast<BoxAxis>(isHorizontal);
}

BoxAxis blockAxis2(bool isHorizontal)
{
    return isHorizontal ? BoxAxis::Horizontal : BoxAxis::Vertical;
}
```

It is thus possible to have the same perf with a ternary than with a static_cast, without the code being as fragile.